src/ImageSharp/Formats/Jpeg/JpegDecoderCore.cs

```diff
@@ -751,7 +746,7 @@ private void ProcessApplicationHeaderMarker(BufferedReadStream stream, int remai
             stream.Read(this.temp, 0, JFifMarker.Length);
             remaining -= JFifMarker.Length;
 
-            JFifMarker.TryParse(this.temp, out this.jFif);
+            _ = JFifMarker.TryParse(this.temp, out this.jFif);
```

Comment:
Should we not handle this?

Reply:
Yeah, you're right. Now explicitly throwing `NotSupportedException` since we only support JFIF App0 markers.